webapp/src/components/Competency.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+interface CompetencyProps {
+  label: string;
+  description: string;
+}
+
+const Competency = ({ label, description }: CompetencyProps) => (
+  <div>
```

Comment:
I guess it would be out of the scope of this ticket to have it styled with Material UI?

Reply:
Yes.  We should style with Material UI in a later issue.